gamemodes/terrortown/entities/weapons/weapon_ttt_phammer.lua

```diff
@@ -0,0 +1,416 @@
+---
+-- @class SWEP
+-- @section weapon_ttt_phammer
+
+AddCSLuaFile()
+
+DEFINE_BASECLASS "weapon_tttbase"
+
+SWEP.HoldType              = "ar2"
+
+if CLIENT then
+	SWEP.PrintName          = "polter_name"
+	SWEP.Slot               = 7
+
+	SWEP.ViewModelFlip      = false
+	SWEP.ViewModelFOV       = 54
+
+	SWEP.EquipMenuData = {
+		type = "item_weapon",
+		desc = "polter_desc"
+	};
+
+	SWEP.Icon               = "vgui/ttt/icon_polter"
+end
+
+SWEP.Base                  = "weapon_tttbase"
+
+SWEP.Primary.Recoil        = 0.1
+SWEP.Primary.Delay         = 12.0
+SWEP.Primary.Cone          = 0.02
+SWEP.Primary.ClipSize      = 6
+SWEP.Primary.DefaultClip   = 6
+SWEP.Primary.ClipMax       = 6
+SWEP.Primary.Ammo          = "Gravity"
+SWEP.Primary.Automatic     = false
+SWEP.Primary.Sound         = Sound( "weapons/airboat/airboat_gun_energy1.wav" )
+
+SWEP.Secondary.Automatic   = false
+
+SWEP.Kind                  = WEAPON_EQUIP2
+SWEP.CanBuy                = {ROLE_TRAITOR} -- only traitors can buy
+SWEP.WeaponID              = AMMO_POLTER
+
+SWEP.UseHands              = true
+SWEP.ViewModel             = "models/weapons/c_irifle.mdl"
+SWEP.WorldModel            = "models/weapons/w_IRifle.mdl"
+
+SWEP.NoSights              = true
+
+SWEP.IsCharging            = false
+SWEP.NextCharge            = 0
+SWEP.MaxRange              = 800
```

Comment:
just FYI, in our codebase we don't use those aligned `=`. But you can keep if for now